server/src/main/java/io/druid/segment/loading/SegmentLoaderLocalCacheManager.java

```diff
@@ -46,10 +48,18 @@
   private final SegmentLoaderConfig config;
   private final ObjectMapper jsonMapper;
 
-  private final SortedSet<StorageLocation> locations;
+  private final List<StorageLocation> locations;
```

Comment:
may be add a note here that use `getSortedList(locations)` anytime someone wants to access `locations`